tests/robustness/multiclient_test/storagestats/storage_stats.go

```diff
@@ -25,54 +25,44 @@ const (
 
 var logFilePath string
 
-// DirDetails represents details about a directory,
+// DirectorySize represents details about a directory,
 // path, and size.
-type DirDetails struct {
-	DirPath string `json:"dirPath"`
-	DirSize int64  `json:"dirSize"`
+type DirectorySize struct {
+	Path string `json:"Path"`
+	Size int64  `json:"Size"`
```

Comment:
nit: this is the convention
```suggestion
	Path string `json:"path"`
	Size int64  `json:"size"`
```